### PR TITLE
Fixes #23411 - use dynflowd in services for foreman-tasks

### PR DESCRIPTION
**Pull request by johnpmitsch**
In foreman 1.17+, foreman-tasks was renamed to dynflowd

### CODE REVIEW

**Comment by jturel** [comment]
@johnpmitsch could you add a description of what an acceptance test for this change would be for someone who has never used foreman_maintain before? :)

**Comment by johnpmitsch** [comment]
@jturel yes, there is a reproducer in the issue. The main thing is after you disable and enable services you can restart tasks fine.

From the foreman_maintain directory:
```
./bin/foreman-maintain service disable
./bin/foreman-maintain service enable
./bin foreman-maintain service restart
```

This should work on katello/satellite systems using foreman > 1.17 and < 1.17 (that is where the service name changed) so its worth checking both of these cases (can use older Katello or Satellite to check < 1.17)

File: lib/foreman_maintain/concerns/system_helpers.rb [hunk 1]
```diff
@@ -56,6 +56,11 @@ def execute?(command, input = nil)
         $CHILD_STATUS.success?
       end
 
+      def rpm_greater_than_version?(name, target_version)
```
Comment (jturel) [inline lib/foreman_maintain/concerns/system_helpers.rb]: nitpick: this method is checking >= not just > so maybe `rpm_gte_version` ?

**Comment by jturel** [comment]
Works nicely. I tested 3.6 and 3.5. Left one comment that you may or may not want to address, but I will ACK it now!

File: lib/foreman_maintain/concerns/system_helpers.rb (hunk 1 above)
Comment (jturel) [reply lib/foreman_maintain/concerns/system_helpers.rb]: i'm good with that!

Comment (johnpmitsch) [reply lib/foreman_maintain/concerns/system_helpers.rb]: updated the method name :+1:

Comment (jturel) [reply lib/foreman_maintain/concerns/system_helpers.rb]: still works - ship it 👍

**Comment by johnpmitsch** [comment]
@iNecas of course that already exists :smile: updated to use `check_min_version`

**Comment by iNecas** [comment]
Thanks @johnpmitsch

**Pull request merged**